doc/tutorial/single_cell_recipe.rst

```diff
@@ -38,9 +42,15 @@ and overrides and implements some of :class:`arbor.recipe` methods. Not all meth
 have to be overridden, but some will always have to be, such as :meth:`arbor.recipe.num_cells`.
 It returns `0` by default and models without cells are quite boring!
 
+.. note::
```

Comment:
I find this note  confusing. What's the argument being made? I assume it is this:
> A recipe must derive from `arbor.recipe` and must implement a number of methods (documented [there]).
In addition some methods may optionally be overridden. Beyond this, it is up to you, the user, to structure your code as you find convenient.

Also: `inherited` is not a word that can be used here. You can `inherit from` or have a `derived class`. In OOP that is.

Reply:
I reworded this.